src/UICrafter.Mobile/Components/Pages/AppViewPage.razor

```diff
@@ -19,31 +19,17 @@
 
 @if (_isLoading)
 {
-    <MudProgressCircular Color="Color.Primary" Indeterminate="true" />
+    <div style="display: flex; justify-content: center; align-items: center; height: 100vh;">
```

Comment:
This essentially what MudOverlay is for where you add the fixed or absolute property depending on what you want.